frame/transaction-payment/src/lib.rs

```diff
@@ -533,14 +540,93 @@ mod tests {
 				RuntimeDispatchInfo {
 					weight: info.weight,
 					class: info.class,
-					partial_fee: (
+					partial_fee:
 						5 /* base */
-						+ len as u64 /* len * 1 */
-						+ info.weight.min(MaximumBlockWeight::get()) as u64 * 2 /* weight * weight_to_fee */
-					) * 3 / 2
+						+ (
+							len as u64 /* len * 1 */
+							+ info.weight.min(MaximumBlockWeight::get()) as u64 * 2 /* weight * weight_to_fee */
+						) * 3 / 2
 				},
 			);
 
 		});
 	}
+
+	#[test]
+	fn compute_fee_works() {
```

Comment:
I recommend the test to be broken into smaller ones, but up to you regarding how to do it. it is a bit too lengthy. Maybe: `compute_fee_works_with_multipier`, `compute_fee_works_without_multiplier`?